Reference new contributing guidelines

MisRob opened the pull request:
## Summary

Reference new contributing guidelines and remove obsolete or duplicate information

## Reviewer guidance

- Can you think of some places I forgot to update?

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Documentation**
  - Simplified contribution instructions in the README and removed the CONTRIBUTING.md file, replacing detailed steps with a welcoming statement and a link to an external contribution guide.
  - Updated developer documentation to remove outdated support channels and direct contributors to a centralized external resource for contribution information.
  - Added prominent headers to issue templates clarifying that certain issues are not open for contribution and directing users to contribution guidelines.
  - Disabled creation of blank issues via GitHub configuration.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

MisRob committed:
Reference new contributing guidelines

rtibbles commented on docs/contributing/ways_to_contribute.rst:
```diff
@@ -36,7 +34,7 @@ Please make sure to use the template. Replace the HTML comments (the ``<!-- ...
 ----------
 
-See `How can I contribute? <https://github.com/learningequality/kolibri/blob/develop/CONTRIBUTING.md>`__
+Visit the `Contributing to our open code base <https://learningequality.org/contributing-to-our-open-code-base/>`__
```
I don't think `the` is necessary here (or in the other two references) can just say `Visit <page title>`.

rtibbles reviewed (changes requested):
Slight wording revision needed.

MisRob commented:
Thanks @rtibbles. I removed 'the' in https://github.com/learningequality/.github/pull/25. After we merge that, we can see how it works and then I will complete this PR.

rtibbles commented:
That PR has been merged!

MisRob committed:
Remove CONTRIBUTING.md

In favour of the default .github/.github/CONTRIBUTING.md

MisRob committed:
Fix grammar

MisRob committed:
Add header to issue templates

Also disables the default blank issue in favor
of the new custom blank issue template to allow
it to have the header too.

MisRob commented:
@rtibbles https://github.com/learningequality/.github/pull/25 works as expected, one example of being it referenced correctly is at https://github.com/learningequality/test-actions/contribute.

I updated this PR to
- Remove CONTRIBUTING.md
- Remove 'the' from README.md
  - Also  aligned automated header messages in https://github.com/learningequality/.github/pull/26
- Added the default header to issue templates
  - Also disabled the default blank issue in favor of the new custom blank issue template to allow it to have the header too

After we merge this and https://github.com/learningequality/.github/pull/26, I will update remaining repositories and also open pull requests to turn on the header automation.

rtibbles reviewed (approved):
Makes sense to me.

rtibbles commented:
I have approved the PRs, feel free to merge when appropriate!
